src/bzmesh.jl

```diff
@@ -90,3 +90,17 @@ function bzmesh_ir_wedge(kgrid_size, lattice, composition...; tol_symmetry=1e-5)
 
     kcoords, ksymops
 end
+
+@doc raw"""
+Selects a kgrid_size to ensure a minimal spacing (in inverse Bohrs) between kpoints.
+Default is ``2π * 0.04 \AA^{-1}``.
+"""
+function kgrid_size_from_minimal_spacing(lattice, spacing=2π * 0.04 / units.Ǎ)
+    for d in 1:3
+        @assert norm(lattice[:, d]) != 0
```

Comment:
You don't need that, the following line will fail anyway

Reply:
Yeah with the very helpful `SingularException` :-P . I think getting the assertion and then finding the line with the explanatory comment is more helpful. But I can remove it if you want.

Reply:
Hehe, as usual, I tend towards less code = better. But I'm fine with either